home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/quartz/jobs/SetOverdueStatusForInvitationsJob.java

```diff
@@ -0,0 +1,29 @@
+package com.softserveinc.ita.homeproject.homeservice.quartz.jobs;
+
+import com.softserveinc.ita.homeproject.homeservice.quartz.config.QuartzJobBeanAutoConfiguration;
+import com.softserveinc.ita.homeproject.homeservice.service.impl.ApartmentInvitationServiceImpl;
+import com.softserveinc.ita.homeproject.homeservice.service.impl.CooperationInvitationServiceImpl;
+import lombok.NoArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.quartz.JobExecutionContext;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.quartz.QuartzJobBean;
+import org.springframework.stereotype.Component;
+
+@Component
+@NoArgsConstructor
+@Slf4j
```

Comment:
I can't see the usages of the logger. Do we really need it?

Reply:
Indeed, the logs when executing jobs are the same - either using @ Slf4j, or not. That is, we really do not need this annotation. True, I do not understand where the logging comes from here at all.